data/mods/Magiclysm/monsters/monsters.json

```diff
@@ -359,5 +359,39 @@
     "death_function": [ "NORMAL" ],
     "regenerates": 1,
     "flags": [ "SEES", "SMELLS", "HEARS", "STUMBLES", "PATH_AVOID_FIRE", "REVIVES" ]
+  },
+  {
+    "id": "mon_loup_garou",
+    "type": "MONSTER",
+    "name": { "str": "loup garou", "str_pl": "loup garous" },
+    "description": "Previously thought extinct outside of rural Quebec and Louisiana this cursed creature has been known to breed with true wolves when it isn't madly destroying any signs of humanity in it's territory.",
+    "default_faction": "wolf",
+    "bodytype": "dog",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "60 L",
+    "weight": "100 kg",
+    "hp": 120,
+    "speed": 145,
+    "material": [ "flesh" ],
+    "symbol": "w",
+    "looks_like": "mon_wolf",
+    "color": "light_gray",
+    "morale": 20,
+    "melee_skill": 7,
+    "melee_dice": 5,
+    "melee_dice_sides": 4,
+    "melee_cut": 4,
+    "dodge": 14,
+    "armor_bash": 48,
+    "armor_cut": 36,
+    "armor_bullet": 40,
+    "vision_night": 5,
+    "harvest": "mammal_fur",
+    "path_settings": { "max_dist": 10 },
+    "anger_triggers": [ "STALK", "FRIEND_ATTACKED", "FRIEND_DIED", "PLAYER_WEAK", "PLAYER_CLOSE" ],
+    "death_function": [ "NORMAL" ],
+    "zombify_into": "mon_zolf",
```

Comment:
This might deserve its own zombie version

Reply:
I think you are right on this.

Reply:
Added.  Let me know what you think.